bugbug/models/triage.py

```diff
@@ -0,0 +1,315 @@
+# -*- coding: utf-8 -*-
```

Comment:
Given that this is implementing the same as the components model but with a different ML algorithm, maybe it's better to call the file something like `component_nn.py` (nn as in neural networks).